cli/cmd/generate_azure.go

```diff
@@ -0,0 +1,680 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/imdario/mergo"
+	"github.com/spf13/cobra"
+
+	"github.com/lacework/go-sdk/lwgenerate/azure"
+	"github.com/pkg/errors"
+)
+
+var (
+	// Define question text here so they can be reused in testing
+	QuestionAzureEnableConfig = "Enable Azure Config Integration?"
+	QuestionAzureConfigName   = "Specify Name of the Config"
+	QuestionEnableActivityLog = "Enable Azure Activity Log Integration?"
+	QuestionActivityLogName   = "Specify name of the Activity Log"
+
+	QuestionAzureAnotherAdvancedOpt      = "Configure another advanced integration option"
+	QuestionAzureConfigAdvanced          = "Configure advanced integration options?"
+	QuestionAzureCustomizeOutputLocation = "Provide the location for the output to be written:"
+
+	// Active Directory
+	QuestionEnableAdIntegration = "Create Active Directory Integration?"
```

Comment:
nitpick: This feels like we should be asking the inverse here. `Use existing Active Directory integration?`
It also feels like an "advanced option" IMO

Reply:
I see what you mean, just that in the case where they do not use an existing active directory integration we need to create one for them, should they user not have that called out rather than it happening by default because they do not have details of a current one

Reply:
IMO it's a bit redundant to ask this. What if they answer no to `Use existing Active Directory integration?` and this?

Reply:
I think we need to ask the question to let them know that active directory integration will be created if they answer no to this an no to `Use existing Active Directory integration?` then the error` invalid inputs: Active directory details must be set `will be shown to the user